src/js/media/views/toolbar.js

```diff
@@ -144,7 +144,7 @@ Toolbar = View.extend(/** @lends wp.media.view.Toolbar.prototype */{
 
 			var requires = button.options.requires,
 				disabled = false,
-				modelsUploading = ! _.isEmpty( library.findWhere(  { 'uploading': true } ) );
+				modelsUploading = 'undefined' !== typeof library && ! _.isEmpty( library.findWhere( { 'uploading': true } ) );
```

Comment:
```suggestion
				modelsUploading = library && ! _.isEmpty( library.findWhere( { 'uploading': true } ) );
```

Not blocker.

Similar to other places can we check `library` directly?

Reply:
I think I tried that first and got an undefined error in this case, but I will try again,

Reply:
Actually that does work correctly @mukeshpanchal27, thanks! Updated.